bin/API_Migration_scripts_readme/Methods in Java Files.ipynb

```diff
@@ -0,0 +1,384 @@
+   "metadata": {},
+   "source": [
+    "# Methods in Plugins"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab88f421",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#imports\n",
+    "import os\n",
+    "import csv\n",
+    "import requests"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6694d43d",
+   "metadata": {},
+   "source": [
+    "## Python Script to check methods"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e2dcb10e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def find_matches(csv_file,java_directory): #custom method\n",
+    "    \n",
+    "    methods = []\n",
+    "    with open(csv_file,\"r\") as csvfile: #reading the CSV\n",
+    "        reader = csv.reader(csvfile, delimiter=\",\")\n",
+    "        for row in reader:\n",
+    "            method_name = row[0]\n",
+    "            methods.append(method_name)\n",
+    "            \n",
+    "    matching_methods = [] #reading the java file\n",
+    "    for file in os.listdir(java_directory):\n",
+    "        if file.endswith(\".java\"):\n",
+    "            with open(os.path.join(java_directory, file), \"r\") as f:\n",
+    "                for line in f:\n",
+    "                    for method in methods:\n",
+    "                        if method in line:\n",
+    "                            matching_methods.append((file, line))\n",
+    "                            \n",
+    "    return matching_methods"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab5108dd",
+   "metadata": {},
+   "source": [
+    "### 1. Data Collection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d129d6d5",
+   "metadata": {},
+   "source": [
+    "#### Main Folder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "148c0e2b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Absolute PATH"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ebbbc031",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if __name__ == \"__main__\":\n",
+    "    csv_file = \"/tmp\"\n",
+    "    java_directory = \"/tmp\"\n",
+    "    matching_methods = find_matches(csv_file, java_directory)\n",
+    "    for file, line in matching_methods:\n",
+    "        print(f\"{file} has {line}\") #to print which method is in which file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82813869",
+   "metadata": {},
+   "source": [
+    "#### Location"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a842f54c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if __name__ == \"__main__\":\n",
+    "    csv_file = \"/tmp\"\n",
+    "    java_directory = \"/tmp\"\n",
+    "    matching_methods = find_matches(csv_file, java_directory)\n",
+    "    for file, line in matching_methods:\n",
+    "        print(f\"{file} has {line}\")"
+   ]
+  },
```

Comment:
I don't understand:
- why we need multiple copies of these lines? and
- why all of them have the `java_directory` as `/tmp`?

if you have the same inputs and are running the same script, you will get the same outputs